### PR TITLE
[Bugfix:Submission] Moved the text outside the label tag.

### LINKED ISSUE #10039: Easy to misclick "Hide from Students"
**Describe the bug**
The "Hide from Students" option in course materials is a rarely used feature, but it is easy for the instructor to misclick this button and not realize what happened.

Clicking anywhere on the paragraph of text beneath the checkbox and to the right of the "hide from students" text toggles this checkbox:
Screenshot 2023-12-02 at 9 32 26 PM

It's too easy to click this (especially when an instructor is clicking away from the date picker to close the date picker popup).

**Expected behavior**
Let's reduce the active area for toggling that checkbox to just the checkbox. 

[ tested on mac chrome.  not sure if all other OS & browsers do the same thing ]

### PR DESCRIPTION
**Pull request by sahilsuman933**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant
* [X] Screenshots are attached to Github PR if visual/UI changes were made

### What is the current behavior?
Fixes #10039 

On clicking anywhere on the paragraph of text beneath the checkbox and to the right of the "hide from students" text toggles this checkbox.

image

### What is the new behavior?
=> On clicking anywhere on the paragraph of text beneath the checkbox and to the right of the "hide from students" text will not toggle this checkbox.

image

### Other information?

None.

### CODE REVIEW

**Review by cjreed121 — CHANGES_REQUESTED** [review]
Thanks for the contribution! According to your screenshot, there seems to be a larger gap between the button and text which shouldn't change. Could you use some CSS or maybe switch the p to span so that the gap doesn't change?

File: site/app/templates/course/EditCourseMaterialsFolderForm.twig [hunk 1]
```diff
@@ -55,9 +55,12 @@
         <label>
             <input id="hide-folder-materials-checkbox-edit" type="checkbox" onchange="handleHideMaterialsChange()">
             Hide from Students<br>
+
```
[the remark this answers is not in the document]
Comment (sahilsuman933) [reply site/app/templates/course/EditCourseMaterialsFolderForm.twig]: @cjreed121 I have removed the extra line, <br> tag and fixed the gap issue.

**Review by cjreed121 — CHANGES_REQUESTED** [review]
Thanks for making the change. One more change and this should be good to merge. Before this PR, the red warning text appears below the note. After this PR, the red warning text appears above the note. Can you change the bit of JavaScript responsible for that to ensure it stays in the same spot?

**Comment by sahilsuman933** [comment]
@cjreed121 I am aware of this, but I believe it would be more appropriate to display the warning as a note before the text. It would provide a better user experience. If you still want me to make the change, please let me know and I will fix it.

**Review by cjreed121 — CHANGES_REQUESTED** [review]
Thanks for changing that back. It looks like the gap is still there. I think the label also needs to margin removed. And then the gap between the note and warning as well (they should all be right next to each other). Can you update that? After that this should be good.
image

**Comment by sahilsuman933** [comment]
@cjreed121 Is the margin okay now? 
image

**Review by cjreed121 — APPROVED** [review]
Looks good to me. Thanks for making the changes.

**Pull request merged**